components/raftstore/src/store/snap.rs

```diff
@@ -1982,7 +1982,16 @@ impl TabletSnapManager {
 
     pub fn total_snap_size(&self) -> Result<u64> {
         let mut total_size = 0;
-        for entry in file_system::read_dir(&self.base)? {
+        let dirs = file_system::read_dir(&self.base);
+        if dirs.is_err() {
```

Comment:
Why ignore this error?

Reply:
I think it may have conflict with gc logic. 
It may run into error, like file or directory not found.

Reply:
Not found errors are ignore explicitly for entries. For the directory itself, it's created when the struct is created, so it can't not found.